Makefile

```diff
@@ -275,6 +276,11 @@ docs-other:
 docs-serve: docs ## Run the aurae.io static website locally
 	sudo -E ./hack/serve.sh
 
+.PHONY: docs-test
+docs-test: docs/crate/auraed/ docs/crate/aer/ docs/crate/aurae_client docs/crate/auraescript
```

Comment:
This should consistently have a slash or no slash and probably test more file paths and directories.

Not sure if anyone has better ideas for how to test they've been generated.

The error message could definitely also be better, telling folks how to generate the docs it's sad are missing instead of just that some are missing.